src/Lecture1.hs

```diff
@@ -67,7 +72,8 @@ sumOfSquares x y = error "TODO!"
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = abs(rem n 10)
```

Comment:
That's a perfect implementation 🔝 One minor nitpick is a space addition 👾 

```suggestion
lastDigit n = abs (rem n 10)
```

Alternatively, you can use `mod` instead of `rem` and even write it in the infix form of a function by putting it inside backticks :slightly_smiling_face: 
```suggestion
lastDigit n = abs n `mod` 10
```

Both implementations are fine and I don't see a clear winner. So depends on your preferences 🙂